src/bp/lang-server/api.ts

```diff
@@ -133,13 +146,16 @@ export default async function(options: APIOptions) {
   app.post('/vectorize', waitForServiceMw, async (req, res, next) => {
     try {
       const input = req.body.input
-      const language = req.body.lang || 'en'
+
+      const language = req.body.lang
+      assertLanguage(options, language)
```

Comment:
"assertLanguage" doesn't reallly talks by itself, I suggest changing it with assertValidLanguage.

Also, you could move that validation in an express middleware (similar implementation to waitForServiceMw)